util.go

```diff
@@ -25,14 +27,26 @@ func NewTaskRunner(task Task, ctx context.Context) *TaskRunner {
 	}
 }
 
+func (t *TaskRunner) stop() {
```

Comment:
I would like to have a name with an int value other than 0,1. For example, AVAILABLE, DIE, and so on. Another way is to have comments about 0,1

Reply:
That would be great idea!